clients/admin-ui/src/features/common/nav/v2/nav-config.ts

```diff
@@ -109,6 +112,13 @@ export const configureNavGroups = ({
         return;
       }
 
+      if (
+        route.path === "/privacy-requests/configure" &&
+        !hasAccessToPrivacyRequestConfigurations
```

Comment:
So I was thinking something more generic like how `requiresPlus` is an option that can be set on any route. It isn't a big deal when there's just one, but if we start hard-coding route paths it's likely this function will grow to become a mishmash of code and config constants.

Reply:
That's fair - fwiw, this is temporary until I get to [this ticket](https://github.com/ethyca/fides/issues/2455) in the sprint (after Adam merges in the API changes), but I can create an option similar to `requiresPlus`

Reply:
Yeah I'm thinking if FF-flagged routes are going to come up often I can update the nav config logic to make it easier to check them.